xtask/src/main.rs

```diff
@@ -40,6 +84,9 @@ enum TestCommand {
         /// Overwrite the expected output instead of comparing it.
         #[structopt(long)]
         overwrite: bool,
+        /// Runs a single snapshot test in Debug mode
+        #[structopt(long, default_value = "all")]
+        single: Snapshot,
```

Comment:
Being able to pass `--single all` is a bit odd, maybe this would be better as a positional argument? Then you could do `cargo xtask test-snapshot` or `cargo xtask test-snapshot all` to run all of them, or `cargo xtask test-snapshot bitflags` to run a single one

Reply:
Refactored the `single` option into a positional parameter.